linera-service/src/linera.rs

```diff
@@ -745,6 +745,10 @@ enum WalletCommand {
         /// The path to the genesis configuration for a Linera deployment.
         #[structopt(long = "genesis")]
         genesis_config_path: PathBuf,
+
+        // The root chains to initialize.
```

Comment:
Maybe "initialize" is the wrong word: It creates an entry in the wallet, so that this client can follow the chain and e.g. start a node service with it.

Reply:
"follow" ?

Reply:
Also, maybe we can do better that `--chain-ids`. For instance, `--with-external-chains`.

Reply:
Done. Although I'm not sure about the word "external". Maybe `--with-root-chains` would be better?

Reply:
also possible --with-other-chains
is it important that they are root chains?